infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LRRollingUpgradeHandler.java

```diff
@@ -90,13 +99,13 @@ public boolean isLRUpgradeInProgress(TxnContext txnContext) {
      */
     public void migrateData(TxnContext txnContext) {
         // Data migration to be added here.
-        eventSnapshotSyncRollingUpgrade(txnContext);
+        addEventForSnapshotSyncOnRollingUpgrade(txnContext);
     }
 
     /**
      * Add flag to event table to trigger snapshot sync.
      */
-    public void eventSnapshotSyncRollingUpgrade(TxnContext txnContext) {
+    public void addEventForSnapshotSyncOnRollingUpgrade(TxnContext txnContext) {
```

Comment:
nit: can you rename this method to  'addSnapshotSyncEventOnUpgradeCompletion' ?  a bit shorter and clearer...